addon/templates/components/md-nav-link.hbs

```diff
@@ -0,0 +1,6 @@
+<a href={{href}}>
+{{#if icon}}
+  <i class="material-icons {{iconClass}}">{{icon}}</i>
+{{/if}}
+{{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
```

Comment:
Yes, that reads better to me.  Thanks.

> On Jul 23, 2016, at 4:22 PM, Mike North <notifications@github.com> wrote:
> 
> In addon/templates/components/md-nav-link.hbs <https://github.com/mike-north/ember-cli-materialize/pull/292#discussion_r71969419>:
> 
> > @@ -0,0 +1,6 @@
> > +<a href={{href}}>
> > +{{#if icon}}
> > +  <i class="material-icons {{iconClass}}">{{icon}}</i>
> > +{{/if}}
> > +{{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
> This is an effort to keep consistent with {{link-to}}, since this component is intended to be private (I know I could do a lot better being clear about this) and exposed as a contextual component
> 
> {{#md-nav as |n|}}
>   {{n.title 'My App' 'index'}}
>   {{#n.menu class='left' as |m|}}
>     {{!-- some md-nav-link components --}}
>     {{m.link-to 'Thing 1' 'app.myroute' (query-params a="1") icon='star'}}
>     {{m.link-to 'Thing 2' 'app.myroute' (query-params a="2")}}
>   {{/n.menu}}
> {{/md-nav}}
> Under the hood, these are extended {{link-to}} (so we get all the good stuff that comes along with them). Unfortunately, because positional parameters are used in the base class, it would be a lot of work to support what you're asking for
> 
> What you're trying to do might feel more natural as
> 
> {{#m.link-to 'index'}}<i class="material-icons">star</i>{{/m-link-to}}
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mike-north/ember-cli-materialize/pull/292/files/f37c764020066de8df9534e66eaaff52e2b738ba#r71969419>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABCWvYAQCxoYp30js5yIvJlt6meKSDVwks5qYc8pgaJpZM4HvhE3>.
> 

-- 
Dustin Farris